src/index.js

```diff
@@ -36,17 +36,21 @@ export default function (options = {}, callback = function noop () {}) {
           logger.log('info', 'Re-running the following test files:\n')
           logger.log('info', failedSpecs.join('\n') + '\n')
         }
-        return startProtractor(failedSpecs)
+        return startProtractor(failedSpecs, true)
       }
 
       callback(status, output)
     }
   }
 
-  function startProtractor (specFiles = []) {
+  function startProtractor (specFiles = [], retry = false) {
     let output = ''
     let protractorArgs = [parsedOptions.protractorPath].concat(parsedOptions.protractorArgs)
 
+    if (retry) {
+      protractorArgs.push('--params.flake.retry', true)
```

Comment:
A user can pass multiple `--params` arguments. The `params.flake` should avoid name collision.